File: microservices/ms9-gestioncompras/src/main/java/com/compras/model/Product.java
package com.compras.model;

import java.util.List;

import jakarta.json.bind.annotation.JsonbTransient;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.OneToMany;
import jakarta.persistence.OneToOne;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Product {
  @Id
  @GeneratedValue(strategy = GenerationType.UUID)
  String id;

  String name;
  String descr;

  String imageURL;
  
  double price;
  
  @JsonbTransient
  @OneToOne
  Inventory inventory;

  @JsonbTransient
  @OneToMany
  List<Review> reviews;

  @JsonbTransient
  @OneToMany
  List<OrderItem> orderItems;
}

File: microservices/ms11-calificacion-logica/Model/Product.cs
using System.Collections.Generic;

namespace ms11.Model
{

    public class Product
    {
        public required string id { get; set; }
        public string name { get; set; }
        public string descr { get; set; }
        public string imageURL { get; set; }
        public double price { get; set; }
    }

}